xml/System/Int32.xml

```diff
@@ -2524,12 +2523,12 @@
         <Parameter Name="result" Type="System.Int32" RefType="out" Index="3" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="s">To be added.</param>
-        <param name="style">To be added.</param>
-        <param name="provider">To be added.</param>
-        <param name="result">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="s">A span containing the characters that represent the number to convert.</param>
```

Comment:
This one has a style parameter but isn't referenced like the other descriptions are.